microcosm_logging/factories.py

```diff
@@ -113,20 +112,17 @@ def make_json_formatter(graph):
     Used for searchable aggregation.
 
     """
-    fields = {
-        # what was logged?
+    fields = """{
         "loggerName": "%(name)s",
         "levelName": "%(levelname)s",
-        "message": "%(message)s",
-        # when was it logged?
+        "message": %(message)s,
```

Comment:
We could also add more defensive logic here to support both types, but that seems annoying.